services/errorHandler.js

```diff
@@ -0,0 +1,9 @@
+exports.handleError = function (err, res) {
```

Comment:
Since we're doing logic specific to the error that's returned by mongoose for user creation, can we call this something like "handleMongooseError".

Reply:
good point, changed